### PR TITLE
SEO-183463 ASP.NET Missing H1

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,
Have worked on missing H1, please review.
Regards,
Yvone.

### CODE REVIEW

File: aspnet/Grid/How-to/Custom-Binding-for-Grid-CRUD-operation.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-description: custom binding for grid crud operation
+title: Custom binding in ASP.NET Web Forms Grid Control | Syncfusion
+description: Learn here all about custom binding for grid crud operation in Syncfusion ASP.NET Web Forms Grid control, its elements and more.
 platform: aspnet
 control: Control Name undefined
 documentation: ug
 ---
 
-### Custom Binding for Grid CRUD operation
+# Custom Binding for Grid CRUD operation
```
Comment (gowthamanthavasiyappan) [inline aspnet/Grid/How-to/Custom-Binding-for-Grid-CRUD-operation.md:10]: Is this a proper description? where is the platform and control names?
image

File: aspnet/PdfViewer/Supported-and-Unsupported-Elements.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+title: Supported and unsupported elements in ASP.NET PDF Viewer | Syncfusion
+description: Learn here all about supported and unsupported elements in Syncfusion Essential ASP.NET PDF Viewer control, its elements and more.
 platform: aspnet
 control: PDF viewer
 documentation: ug
 keywords: PDF viewer, pdfviewer
 ---
 
-## Supported and Unsupported Elements
+# Supported and Unsupported Elements
```
Comment (gowthamanthavasiyappan) [inline aspnet/PdfViewer/Supported-and-Unsupported-Elements.md:11]: Is this a proper description? where is the platform and control names?

**Review by gowthamanthavasiyappan — COMMENTED** [review]
Commented

File: aspnet/PdfViewer/Supported-and-Unsupported-Elements.md (hunk 2 above)
Comment (Yvone-Atieno) [reply aspnet/PdfViewer/Supported-and-Unsupported-Elements.md:11]: changed

File: aspnet/Grid/How-to/Custom-Binding-for-Grid-CRUD-operation.md (hunk 1 above)
Comment (Yvone-Atieno) [reply aspnet/Grid/How-to/Custom-Binding-for-Grid-CRUD-operation.md]: changed

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]

**Pull request merged**